docs/modules/ROOT/pages/migration.adoc

```diff
@@ -479,6 +479,13 @@ class WebSocketSecurityConfig: WebSocketMessageBrokerConfigurer {
 ----
 ====
 
+[[servlet-opt-in-sha256-rememberme]]
+==== Use SHA-256 in Remember Me
+
+The `TokenBasedRememberMeServices` implementation now supports SHA-256 for the Remember Me token and this is the default in Spring Security 6.
+You can start using it in 5.8 by setting the `encodingAlgorithm` property to SHA-256 while still decoding tokens that are using MD5.
+See the documentation for xref:servlet/authentication/rememberme.adoc#_tokenbasedremembermeservices[TokenBasedRememberMeServices] for more information.
+
```

Comment:
Please show sample configuration inline. You can include a partial like "Optimize Querying of `RequestCache`" does, or you can do them inline like "Use `AuthorizationManager` for Method Security".

It would be helpful to show a before and after where applicable as this can help the Spring Boot migrator team use this as a reference to build recipes. The diff can also be instructive visually for the reader.